libr/bin/p/bin_coff.c

```diff
@@ -356,7 +361,12 @@ static RList *_relocs_list(RBin *rbin, struct r_bin_coff_obj *bin, bool patch, u
 					case COFF_REL_AMD64_REL32:
 						reloc->type = R_BIN_RELOC_32;
 						reloc->additive = 1;
-						ut64 data = sym_vaddr - reloc->vaddr - 4;
+						ut8 offset[4];
+						if (!rbin->iob.read_at (rbin->iob.io, reloc->vaddr, offset, 4)) {
```

Comment:
same here, i think you can just avoid doing the rest of lines in this case if the read cant happen